pushy/src/main/java/com/turo/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -530,6 +535,58 @@ public ApnsPayloadBuilder setThreadId(final String threadId) {
         return this;
     }
 
+    /**
+     * <p>Sets the summary argument for this notification. The summary argument is:</p>
+     *
+     * <blockquote>The string the notification adds to the category’s summary format string.</blockquote>
+     *
+     * <p>For example when defining an UNNotificationCategory, passing a format string like
+     * "%u more messages from %@" to the categorySummaryFormat argument, will produce
+     * "x more messages from {summaryArgument}.</p>
+     *
+     * <p>By default, it says “x more notifications”, no summary argument is included.</p>
+     *
+     * @param summaryArgument the summary argument for this notification
```

Comment:
Can we call out what happens if we set this to `null`?

```
@param summaryArgument the summary argument for this notification; if
{@code null}, the {@code summary-arg} key is omitted from the payload entirely
```

Reply:
Done in cd6eb42.